src/VisualStudio/CSharp/Impl/CSharpVSResources.resx

```diff
@@ -516,4 +516,7 @@
   <data name="When_possible" xml:space="preserve">
     <value>When possible</value>
   </data>
+  <data name="Warn_on_invalid_string_dot_format_calls" xml:space="preserve">
+    <value>Warn on invalid string.Format method calls</value>
```

Comment:
Nit: I'd change this to something else (@kuhlenh to help with words).  Maybe "Report calls to 'string.Format' with invalid placeholders"?

Is "Warn" right - is the severity always Warning?